### PR TITLE
Check run return code from local IEDB install before proceeding

### PR DESCRIPTION
**Pull request by susannasiebert**
Related to #232

### CODE REVIEW

File: pvacseq/lib/call_iedb.py [hunk 1]
```diff
@@ -49,6 +49,8 @@ def main(args_input = sys.argv[1:]):
 
     if args.iedb_executable_path is not None:
         response = run(prediction_class_object.iedb_executable_params(args), stdout=PIPE)
+        if response.returncode == 2:
```
Comment (tmooney) [inline pvacseq/lib/call_iedb.py]: What's special about return code 2?  Should it also fail on other non-zero exit codes--especially if the process was killed by a signal?  (Could we use `run`'s `check` option?)

Comment (susannasiebert) [inline pvacseq/lib/call_iedb.py]: This is a result of IEDB using `OptionParser`. It returns code 2 when `parser.error` is set. See https://docs.python.org/2/library/optparse.html#how-optparse-handles-errors and the `main` function in /gscuser/ssiebert/mhc_i/src/predict_binding.py.

Comment (susannasiebert) [inline pvacseq/lib/call_iedb.py]: I will look into `run`s `check` option as that seems cleaner.

**Comment by jasonwalker80** [comment]
+1

**Pull request merged**